Adyen/Service/Checkout/RecurringService.cs

```diff
@@ -22,40 +22,70 @@
 
 namespace Adyen.Service.Checkout
 {
+    /// <summary>
+    /// RecurringService Interface
+    /// </summary>
+    public interface IRecurringService
+    {
+        /// <summary>
+        /// Delete a token for stored payment details
+        /// </summary>
+        /// <param name="recurringId"><see cref="string"/> - The unique identifier of the token.</param>
+        /// <param name="shopperReference"><see cref="string"/> - Your reference to uniquely identify this shopper, for example user ID or account ID. Minimum length: 3 characters. &gt; Your reference must not include personally identifiable information (PII), for example name or email address.</param>
+        /// <param name="merchantAccount"><see cref="string"/> - Your merchant account.</param>
+        /// <param name="requestOptions"><see cref="RequestOptions"/> - Additional request options.</param>
+        /// <returns><see cref="StoredPaymentMethodResource"/>.</returns>
+        StoredPaymentMethodResource DeleteTokenForStoredPaymentDetails(string recurringId, string shopperReference = default, string merchantAccount = default, RequestOptions requestOptions = default);
```

Comment:
`recurringId`, I believe this called `recurringDetailReference` Let's rename this and keep it consistent.

src: https://docs.adyen.com/online-payments/classic-integrations/api-integration-ecommerce/recurring-payments/disable-stored-details#request

Reply:
The link you sent is classic payments, plus it seems to be quite old (v25) so I think the specs here are right!